src/common/handleTransfer.ts

```diff
@@ -81,7 +89,24 @@ export const handleTransfer = async ({
         version,
         tokenType: mapAvatarTypeToTokenType(avatarType),
         tokenOwner_id,
+        totalSupply,
       });
+    } else {
+      let totalSupply = 0n;
```

Comment:
isn't it odd that you set it to 0 here? if the Token exists?

Reply:
oops correct